src/gh/diffCheck/diffCheck/df_joint_detector.py

```diff
@@ -47,42 +47,115 @@ def _assign_ids(self, joint_face_ids):
 
         return extended_ids
 
-    def run(self):
+    def find_largest_cylinder(self):
```

Comment:
private encapsulation for `_find_largest_cylinder()` since only used inside DFJointDetector